libtenzir/include/tenzir/location.hpp

```diff
@@ -178,4 +178,19 @@ struct as_located {
 template <class T>
 struct is_located : detail::is_specialization_of<located, T> {};
 
+auto trace_panic(into_location trace, auto&& fun) {
+  try {
+    static_cast<void>(trace);
```

Comment:
That shouldn't be necessary.

Reply:
My local clang warns for this, unfortunately.